src/Core/CSharpVB/RuleSetGenerator.cs

```diff
@@ -38,12 +38,11 @@ namespace SonarLint.VisualStudio.Core.CSharpVB
 {
     public class RuleSetGenerator : IRuleSetGenerator
     {
-        private const string SONARANALYZER_PARTIAL_REPO_KEY = "sonaranalyzer-{0}";
-        private const string ROSLYN_REPOSITORY_PREFIX = "roslyn.";
+        private const string SonarAnalyzerRepositoryPrefix = "sonaranalyzer-{0}";
```

Comment:
Miscellaneous - changed the casing to match the style in SLVS (these constants were originally copied from S4MSB).